app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectCreationActivity.kt

```diff
@@ -110,8 +124,16 @@ class Form : AppCompatActivity() {
                 }
             }
         }
+        else if(resultCode == Activity.RESULT_OK && requestCode == REQUEST_TAG_ACCESS){
+            if (data != null) {
+                listTags = data.getStringArrayExtra("tagsList")!!
```

Comment:
It's better to do null check instead of NPE throwing.

Reply:
fixed with null check, and if null I just put back an empty list